File: frontend/types/dto/response/ComidaResponseDTO.ts
export interface ComidaResponseDTO{
    Id_Comida: number;
    Nombre: string;
    Descripcion?: string;
    Calorias: number;
    Proteinas: number;
    Carbohidratos: number;
    Grasas: number;
    Fibra: number;
    Azucares: number;
    Porcion: string;
    Imagen_Url: string;
    Categorias: string[];
}

File: backend/Dtos/response/ComidaResponseDTO.cs
namespace backend.Dtos.response;

public record ComidaResponseDTO(
    int Id_Comida,
    string Nombre,
    string? Descripcion,
    decimal Calorias,
    decimal Proteinas,
    decimal Carbohidratos,
    decimal Grasas,
    decimal? Fibra,
    decimal? Azucares,
    string Porcion,
    string? Imagen_Url,
    List<CategoriaResponseDTO> Categorias
);
